### ISSUE TITLE
Deepslate fence in cyclic shapebuilder - This error crashed my map

### ISSUE THREAD

**Issue by ribolive** [FTBTeam/FTB-Modpack-Issues#5171]
### Modpack

FTB Skies Expert

### Modpack version

1.7.0

### Log Files

[debug.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/15137004/debug.log)
[console.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/15137007/console.log)

### Describe the bug

This error crashed my map, I can't play on it anymore

How did I generate the error: When trying to place a package of deepslate fence into the cyclic block placing machine!

### Steps to reproduce

When trying to place a package of deepslate fence into the cyclic shape builder machine!

### Expected behaviour

the block enters the machine or some message informing that it does not accept

### Screenshots

![Captura de tela 2024-04-27 011333](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/20013833/6ae32625-79b8-404f-9f6e-7e4e5991cbbe)
![Captura de tela 2024-04-27 011443](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/20013833/521eaafb-6112-4397-aeb3-f018631e9105)

**Comment by ribolive** [comment]
If there is any workaround for me to at least come back with the map to play, I would be grateful!

**Comment by isochronous** [comment]
Restore from a backup?

**Comment by ribolive** [comment]
I was able to fix the map here without having to go back bkp and lose any progression. I noticed that even putting normal deepslate in the machine, the game still crashes.
I also noticed that the information in the position log is correct, but the chuck and region references do not match those present in the log (when changing in NBT Studio).
I changed the item (via NBT) inside the machine to standard cobblestone and the game started working again.
